.rubocop.yml

```diff
@@ -1128,7 +1128,7 @@ Style/VariableInterpolation:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#curlies-interpolate'
   Enabled: false
 
-Style/VariableName:
+Naming/VariableName:
```

Comment:
To add the cops added between versions, you'll need to run mry with the target version (https://github.com/pocke/mry#usage): 
`mry --from=0.38.0 --target=0.52.0 .rubocop.yml`